src/PointModel.js

```diff
@@ -0,0 +1,65 @@
+/*
+* @Author: mike
+* @Date:   2016-04-05 17:03:20
+* @Last Modified 2016-04-05
+* @Last Modified time: 2016-04-05 19:46:20
+*/
+
+import BaseModel from './BaseModel'
+import Promise from 'bluebird'
+
+import traverse from 'traverse'
+import _ from "underscore"
+
+function createPoint(values, next) {
+  var latitude = values.latitude
+  var longitude = values.longitude
+  if(!latitude || !longitude) return next()
+  values[this.queryField] = { type: "Point", coordinates: [ longitude, latitude ] }
+  next()
+}
+
+export default BaseModel.extend({
+
+  // Lifecycle
+
+  queryField: 'geoPoint',
+  
+  beforeCreate: createPoint,
+  
+  beforeUpdate: createPoint,
+
+  createGeoIndex: function() {
+    this.native((err, collection) => {
+      var index = {}
+      index[this.queryField] = '2dsphere'
+      collection.ensureIndex(index, () => {})
+    })
+  },
+
+  findNear: function(latitude, longitude, distance=1000, query={}) {
+    return this._geoFind('$near', [longitude, latitude], distance, query)
+  },
+  
```

Comment:
Curious if you think a code comment here would be useful? Or a burden? The routine is short, yet (maybe since I'm not familiar with Mongo's spatial query) a brief description of the parameters here seems like it would be helpful.